GSA/plainlanguage.gov
# Added `meta` viewport for mobile display

## Summary

Adds the meta tag `<meta name="viewport" content="width=device-width, initial-scale=1.0" />` to the base template for setting mobile webpage display.

### Preview

[Link to Preview](https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.sites.pages.cloud.gov/preview/gsa/plainlanguage.gov/nl-fix-performance-metrics/)

### How To Test

1. Visit homepage
2. Open in responsive web viewer or mobile browser
3. Homepage should be rendered for mobile display


**Before**
<img width="696" alt="Screen Shot 2023-07-10 at 3 46 56 PM" src="https://github.com/GSA/plainlanguage.gov/assets/104778659/150e87ac-9e54-4f4c-8fb9-94116a724bce">

**After**
<img width="766" alt="Screen Shot 2023-07-10 at 3 49 41 PM" src="https://github.com/GSA/plainlanguage.gov/assets/104778659/3d344b46-dd9d-45da-931e-1f932f85be8c">


---

### Dev Checklist
